src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/CursorIT.java

```diff
@@ -0,0 +1,317 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.esintgtest;
+
+import com.amazon.opendistroforelasticsearch.sql.utils.StringUtils;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.client.ResponseException;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static com.amazon.opendistroforelasticsearch.sql.esintgtest.TestsConstants.TEST_INDEX_ACCOUNT;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.containsString;
+
+public class CursorIT extends SQLIntegTestCase {
+
+    private static final String JDBC = "jdbc";
+
+    @Override
+    protected void init() throws Exception {
+        loadIndex(Index.ACCOUNT);
+    }
+
+    @Test
+    public void invalidFetchSize() throws IOException {
+        // invalid fetch_size --> negative(-2), non-numeric("hello")
+        // acceptable fetch_size --> positive numbers, even in string form "532.4"
```

Comment:
should these style of comments go outside of the actual test case as a JavaDoc comment?

Reply:
Not necessary for tests, does not affect readability or clarity. Its a best practice for code exposed as public libraries.

Reply:
Done.